tests/conftest.py

```diff
@@ -167,10 +168,13 @@ def dp(_dp):
     _dp.handlers = {}
     _dp.groups = []
     _dp.error_handlers = {}
-    _dp.__stop_event = Event()
-    _dp.__exception_event = Event()
-    _dp.__async_queue = Queue()
-    _dp.__async_threads = set()
+    # For some reason if we setattr with the name mangled, then some tests(like async) run forever,
+    # due to threads not acquiring, (blocking). This adds these attributes to the
+    # __dict__, without raising a TGDeprecation warning.
```

Comment:
if it reads cleaner with raising TGDeprecationwarning, we can just ignore that with pytest.mark.filterwarnings`?

Reply:
I don't understand, it's about the tests getting stuck at async, that's why this was needed.

Reply:
Mh, then I guess the comment just confused me:

> This adds these attributes to the __dict__, without raising a TGDeprecation warning.